src/govuk/components/task-list/_index.scss

```diff
@@ -0,0 +1,103 @@
+@include govuk-exports("govuk/component/task-list") {
+  .govuk-task-list {
+    @include govuk-font($size: 19);
+    margin-top: 0;
+    margin-bottom: 0;
+    padding: 0;
+    list-style-type: none;
+    @include govuk-media-query($from: tablet) {
+      min-width: 550px;
+    }
+  }
+
+  .govuk-task-list__items {
```

Comment:
What element does this class correspond to? I can't seem to see it in the template.